xwiki-platform-core/xwiki-platform-rendering/xwiki-platform-rendering-macros/xwiki-platform-rendering-macro-code/src/main/resources/pygments/code.py

```diff
@@ -39,13 +40,16 @@
 if pygmentLexer:
   pygmentStyle = None
   if style:
-    try:
-      pygmentStyle = get_style_by_name(style)
-    except ClassNotFound:
-      ""
+    if style == "updatedDefault":
```

Comment:
TLDR: I tried to minimize the changes on the Python side, so I added things more on the Java side. This might be heavier, but it'll probably be easier to maintain.

`default` is the value of `style` when it's not defined in the configuration.
___
For the block of logic in DefaultPygmentsParserConfiguration, there was a need to add this logic somewhere. We could have added it at the `code.py` level too. I have no strong reason for including it in Java, except that we have more experience and tools for maintaining and updating it.
I didnt' add any logic on the python side, the only changes are here to make the custom style work.
____

Since our default is customized now, I thought I'd introduce a constant for it. We can definitely get rid of it, but I thought it would be better to define it rather than have a hardcoded string.
Technically we could have just run with using the previous `default` name, but wired it in python to our XWikiStyle. I think it's easier to understand if we make it clear from the start that this is not the tool's default.